src/components/FeedModal/FeedModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { useFormik } from 'formik';
+import IconButton from '@mui/material/IconButton';
+import {
+  Button,
+  Modal,
+  Avatar,
+  CardHeader,
+  CardContent,
+  TextField,
+} from '@mui/material';
+import ClearIcon from '@mui/icons-material/Clear';
+import FeedModalImageUpload from '@components/FeedModal/FeedModalImageUpload/FeedModalImageUpload';
+import {
+  PreviewImage,
+  CardWrapper,
+} from '@components/FeedModal/FeedModal.style';
+
+function FeedModal() {
+  const [open, setOpen] = useState(false);
+  const [previewUrl, setPreviewUrl] = useState('');
+
+  const handleOpen = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const handleImageUpload = (e: React.ChangeEvent<any>) => {
+    if (!e.currentTarget.files[0]) return;
+
+    console.log(e.currentTarget.files[0]);
+    const objectUrl = URL.createObjectURL(e.currentTarget.files[0]);
+    previewUrl && URL.revokeObjectURL(previewUrl);
+    setPreviewUrl(objectUrl);
+
+    formik.setFieldValue('file', e.currentTarget.files[0]);
+  };
+
+  //formik
+  const formik = useFormik({
+    initialValues: {
+      contents: '',
+    },
+    onSubmit: (values, { setSubmitting }) => {
+      setSubmitting(true);
+      console.log(values);
+      setSubmitting(false);
+    },
+  });
+
+  return (
+    <>
+      <Button onClick={handleOpen}></Button>
+      <Modal open={open} onClose={handleClose}>
```

Comment:
모달의 `background` 색상이 `figma`랑 다른 것 같아요!

아래처럼 수정하시면 간단하게 BackDrop 색상을 커스텀 하실 수 있습니당!

```suggestion
      <Modal
        open={open}
        onClose={handleClose}
        BackdropProps={{
          sx: {
            backgroundColor: '#1118271A',
          },
        }}
      >
```

Reply:
[7999614](https://github.com/prgrms-web-devcourse/Team_KPPL_STUTI_FE/pull/28/commits/799961463b79577fc8a810084b61223e4929085d)
스타일 수정 완료했습니다. !